thirdparty/kyaml/runfn/runfn.go

```diff
@@ -310,7 +232,7 @@ func (r RunFns) mergeContainerEnv(envs []string) []string {
 	return declarative.Raw()
 }
 
-func (r RunFns) getFunctionFilters(global bool, fns ...*yaml.RNode) (
+func (r RunFns) getFunctionFilters(fns ...*yaml.RNode) (
```

Comment:
We can get rid of `fns` here and use `r.Functions` directly and also remove `getFunctionsFromFunctions` and other function wrappers.